src/run/docker-compose.ts

```diff
@@ -229,6 +248,43 @@ export class DockerCompose {
         }
     }
 
+    private async runDockerCommands (commandArgs: string[], logHandler: LogHandler, signal?: AbortSignal): Promise<void> {
```

Comment:
Is this used? Showing up as new code, is it new or is this an artefact of GitHub PR?

Reply:
It doesn't look used, I'd delete.